ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -862,13 +863,16 @@ def test_replica_install_after_restore(self):
                            stdin_text=dirman_password + '\nyes')
 
         # re-initialize topology after restore.
-        topo_name = "{}-to-{}".format(master.hostname, replica1.hostname)
+        topo_name = find_segment(master, replica1)
```

Comment:
`find_segment` needs to be modified and accept the suffix as parameter (either `domain` or `ca`).
`topo_name` can be different for each suffix and needs to be set inside the `for topo_suffix` loop.